server/sonar-ce/src/main/java/org/sonar/ce/container/ComputeEngineContainerImpl.java

```diff
@@ -318,6 +319,7 @@ private static void populateLevel4(ComponentContainer container, Props props) {
       XMLProfileParser.class,
       XMLProfileSerializer.class,
       AnnotationProfileParser.class,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "AnnotationProfileParser"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)